postman/collection.json

```diff
@@ -420,7 +430,7 @@
 							"raw": ""
 						},
 						"url": {
-							"raw": "{{url}}/time-events?personId={{person_id}}",
+							"raw": "{{url}}/time-events?personId={{person_id}}&top=10",
```

Comment:
do we need to past the top value?

Reply:
Yes, at moment is necessary as a value of Custom Repository interface. But I think is important for the database refactor as an argument to the top in SQL queries.